granadilla/forms.py

```diff
@@ -20,11 +20,12 @@
 
 import re
 
+from . import models
 from django import forms
 from django.core.files.uploadedfile import SimpleUploadedFile
 from django.utils.translation import ugettext as _
 
-from granadilla.models import LdapContact, LdapUser
+from granadilla.models import LdapContact, LdapDevice, LdapUser
```

Comment:
If you use ``from . import models``, you no longer need to import the models directly: just replace ``LdapContact`` with ``models.LdapContact`` below :)